prompt_tuning/paraphraser.py

```diff
@@ -0,0 +1,28 @@
+import http.client
+import os
+from dotenv import load_dotenv
+import json
+
+def paraphrase(text: str) -> str:
+    load_dotenv()
+    api_key = os.getenv("RAPID_API_KEY")
+
+    conn = http.client.HTTPSConnection("paraphrase-genius.p.rapidapi.com")
```

Comment:
Is this open-sourced/free/better? Can we instead use a paraphrasing model from huggingsface?

Reply:
I'll try to look into it. This is a pretty reliable source though

Reply:
Sure

Reply:
Ill also look into it too, hugging face + more places. Some of may sid must have already seen though.

Reply:
So i saw, for paraphrasing task specifically i coudnt find but NLP based models i found which we can use here hopefully (from hugging face)- 
So i found T5, which is the like NLP based models category i would say. Under this we have
1. google/flan-t5-base
2. google-t5/t5-base (or small, large)

some more t5 models present. but model coming from `paraphrasing` search, have poor/no docs to tell if they r usable, some are other t5 based(but mentioned ones are better).